lib/matplotlib/ticker.py

```diff
@@ -1016,33 +1039,29 @@ def __call__(self, x, pos=None):
         else:
             base = '%s' % b
 
-        if coeff in self.sublabel:
-            if not is_decade and self.labelOnlyBase:
```

Comment:
I guess this `is_decade` and the following ones are fine (see l.1030 for defining the variable this time) but again, I am not sure it is a good thing to borrow the same name as a module level function.